src/style.css

```diff
@@ -1,3 +1,84 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
```

Comment:
Ensure that the 'Roboto' font is properly loaded in your HTML file. This is crucial for the font-family to be applied correctly.